apps/pie-storybook/stories/contribution/testing/browser-tests.mdx

```diff
@@ -6,23 +6,42 @@ import { Meta } from '@storybook/addon-docs';
 
 When testing web components, we aim to verify real behavior in real browsers. We use **Storybook** and **Playwright** to run our tests directly in the browser, rather than relying on simulated environments or mocked DOM APIs. This ensures our components work as users actually experience them.
 
+
 ## Table of Contents
 
-- [Dos and Don’ts](#dos-and-donts)
+- [Dos and Don'ts](#dos-and-donts)
+- [Running browser tests](#running-browser-tests)
 - [Why test in the browser?](#why-test-in-the-browser)
 - [What do we test?](#what-do-we-test)
 - [Testing philosophy](#testing-philosophy)
 - [Test naming conventions and structure](#test-naming-conventions-and-structure)
 
-## Dos and Don’ts
+## Running browser tests
+Browser tests can usually be run with the following command:
+
+**Using Yarn:**
+
+```bash
+yarn test:browsers
```

Comment:
Similar to my comment regarding a11y execution, component tests can also be run independently by doing:

`yarn test:browsers -- --project=component:chrome`

or

`yarn test:browsers --filter=@justeattakeaway/pie-button -- --project=component:chrome`